# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/moz.build
# Commit: 71e63781b38c
# Full Hash: 71e63781b38cbde024a9057553cae96c98e5ecb2
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-02-13 18:21:33
# Regressor Bug: 1604412
# File Overlap Count: 2
# Description:
#   Bug 1604412 - Enable remote backbuffer GDI compositing r=jmathies,jld
#   
#   This change adds new "remote backbuffer" logic when compositing without
#   HW acceleration on Windows (IE compositing through Cairo using the Win32
#   GDI)
# ==============================================================================

diff -r f2088b5ae50d -r 71e63781b38c widget/windows/moz.build
--- a/widget/windows/moz.build	Thu Feb 13 07:13:50 2020 +0200
+++ b/widget/windows/moz.build	Thu Feb 13 03:59:13 2020 +0000
@@ -68,6 +68,7 @@
     'nsWindowGfx.cpp',
     'nsWinGesture.cpp',
     'ProcInfo.cpp',
+    'RemoteBackbuffer.cpp',
     'ScreenHelperWin.cpp',
     'TaskbarPreview.cpp',
     'TaskbarPreviewButton.cpp',
